frontend/packages/ceph-storage-plugin/integration-tests/tests/installFlow.scenario.ts

```diff
@@ -0,0 +1,254 @@
+import { execSync } from 'child_process';
+import { ExpectedConditions as until, browser } from 'protractor';
+import * as crudView from '@console/internal-integration-tests/views/crud.view';
+import {
+  goToOperatorHub,
+  installOperator,
+  InstallCluster,
+  searchInOperatorHub,
+  uninstallOperator,
+  closeOperatorInstall,
+  storageClusterLink,
+  btnCreate,
+  selectWorkerRows,
+  status,
+  NS,
+  operatorsPage,
+  ocsOperator,
+  ocsOperatorStatus,
+  filterInput,
+  getDataFromRowAndCol,
+  podNameFilter,
+  goToStorageClasses,
+  checkIfClusterIsReady,
+  getPodData,
+  getPodPhase,
+  getPodName,
+} from '../views/subscriptionFlow.view';
+
+const installer = new InstallCluster(NS);
+
+const ocsDeploy = `
```

Comment:
This should be done before Integration Test starts. Why are we setting up infra here?

Reply:
Someone needs to handle this. Is the pipeline going to handle this? Even if it does what would happen if we let them run. They would basically fail with something like a resource already exists. But the integration test will carry on?